### ISSUE TITLE
Add aioax25 0.0.12 as a distribution package test

### ISSUE THREAD

**Issue by diazona** [diazona/setuptools-pyproject-migration#105]
This package exposes the issue where the long description is specified only in `setup.py` and there is no `setup.cfg`. (#102) We should add a distribution package test to check that the plugin can handle that.

Based on [a comment on PR #103](https://github.com/diazona/setuptools-pyproject-migration/pull/103#pullrequestreview-1669762001)

**Comment by diazona** [comment]
I gave this a try thinking it'd be quick, but ran into a very weird error (well, after first realizing that there is no version `0.0.12` so I backed up to `0.0.11`):

```console
$ pip download --no-binary :all: --no-deps aioax25==0.0.11
ERROR: Could not find a version that satisfies the requirement aioax25==0.0.11 (from versions: 0.0.11-signed, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10)
ERROR: No matching distribution found for aioax25==0.0.11
```

@sjlongland any idea what this `0.0.11-signed` business is about?

(not that I will be able to do anything about it for a few days, probably... I'm leaving this issue unassigned for the moment so that someone else can take it if desired)

**Comment by sjlongland** [comment]
Good question… I know I had problems uploading detached OpenPGP signatures of builds once, maybe it's a quirk of that.  But, if there's no 0.0.12, I better fix that. :-)

**Comment by diazona** [comment]
Ahh, I see. Well, I suppose it should be possible to just use `0.0.11-signed` for the test; I don't _think_ anything requires those version numbers to comply with PEP 440, it only matters that pip can download the package. Or if you wanted to reupload it as something like `0.0.11.post0`, we can use that.

**Comment by sjlongland** [comment]
`twine` still had the pre-token authentication credentials loaded.  No good with 2FA, you get sweet-FA.  So after some security token wrangling… (including finding out Pypi won't talk to NitroKeys)

https://pypi.org/project/aioax25/0.0.11.post0/

**Comment by diazona** [comment]
Thanks! Now that I got a moment, I gave it another try, but pip complained that no wheel was available - that's fine though, this will give me a good excuse to update the test infrastructure to handle that case. It should be able to build a wheel locally, or do something equivalent to generate the core metadata. (This just means it'll take me some time to implement, but no problem, this is not a blocker.)

**Comment by sjlongland** [comment]
Yeah, I guess I go for KISS principle here… the source code is small and so never saw the point in building a "wheel".

Much of the time this code runs on PocketBeagle single board computers.  Sometimes a Raspberry Pi.  Both of these targets are ARMv7.  Github however uses AMD64, as does my laptop that I use for developing `aioax25`.
